CONTRIBUTING.md

```diff
@@ -121,7 +121,12 @@ The contents of the repository are structured as follows:
 /CONTRIBUTING.md           # this file
 /LICENSE.txt               # MIT
 /README.md                 # master markdown for GitHub, npmjs.com, npms.io, yarn and others
-/build/                    # a folder for the generated files
+/build/                    # build scripts
+/build/export-exchanges.js # used to create tables of exchanges in the docs during the build
+/build/transpile.js        # the transpilation script
+/build/update-badges.js    # a JS script to update badges in the README and in docs
+/build/vss.js              # reads single-sourced version from package.json and writes it everywhere
+/dist/                     # a folder for the generated browser bundle of CCXT
```

Comment:
Have you checked the python build process? There might be a conflict with `dist` being reserved for python `bdist` and `sdist`.

Reply:
I checked the `npm run build` and `setup.py` scripts, it works. Seems that it Python creates a `dist` folder inside of the `python` folder (`python/dist`), not in the root dir...